lib/Elastica/Util.php

```diff
@@ -197,6 +197,6 @@ public static function convertRequestToCurlCommand(Request $request)
      */
     public static function debugEnabled()
     {
-        return defined('DEBUG') && DEBUG;
+        return defined('ELASTICA_DEBUG') && ELASTICA_DEBUG;
```

Comment:
Agree. I'm not even sure if "debug" is the correct name. Perhaps it is more "verbose" or something like that.